src/main/java/io/cdap/e2e/pages/actions/CdfConnectionActions.java

```diff
@@ -366,4 +374,27 @@ public static void verifyConnectionErrorMessage(String errorMessageLocation) {
     AssertionHelper.verifyElementDisplayed(CdfConnectionLocators.locateConnectionFooterError);
     AssertionHelper.verifyElementContainsText(CdfConnectionLocators.locateConnectionFooterError, expectedErrorMessage);
   }
+
+  /**
+   * Click on the Plus Green Button
```

Comment:
As discussed in the morning call the methods should be in Studio Actions class, move these methods there and make sure there is no change in this class.

Reply:
resolved